### PR TITLE
docs: update table of contents

### PR DESCRIPTION
**Pull request by lirantal**
Updating the table of contents to include the new `Malicious Maintainer` type of compromise.

### CODE REVIEW

File: supply-chain-security/compromises/compromise-definitions.md [hunk 1]
```diff
@@ -1,61 +1,90 @@
...
 * [Trust and Signing](#trust-and-signing)
+* [Malicious Maintainer](#malicious-maintainer)
 * [Attack Chaining](#technique-attack-chaining)
 
-
 ## Dev Tooling
-Occurs when the development machine, SDK, toolchains, or build kit have been exploited.  These exploits often result in the introduction of a backdoor by an attacker to own the development environment.
 
-- _Mitigation_ - Use of trusted binary repositories. Verification of provenance (i.e. signatures) and/or integrity (i.e. checksums) of developer tooling downloads. Bootstrapping development toolchain from a minimal, trusted and auditable seed (ideally source code).
+Occurs when the development machine, SDK, toolchains, or build kit have
```
Comment (PushkarJ) [inline supply-chain-security/compromises/compromise-definitions.md:20]:
```suggestion
Occurs when the development machine, SDK, tool chains, or build kit have
```

Comment (PushkarJ) [reply supply-chain-security/compromises/compromise-definitions.md:20]: This is mostly for spellcheck

File: supply-chain-security/compromises/compromise-definitions.md [hunk 2]
```diff
@@ -1,61 +1,90 @@
...
-Occurs when the signing key used is compromised, resulting in a breach of trust of the software from the open source community or software vendor.  This kind of compromise results in the legitimate software being replaced with a malicious, modified version.
 
-- _Mitigation_ - Follow [best practices](https://www.entrustdatacard.com/knowledgebase/best-practices-for-code-signing-certificates) regarding code signing key protection.  A more indepth explanation can be found from [NIST](https://csrc.nist.gov/CSRC/media/Publications/white-paper/2018/01/26/security-considerations-for-code-signing/final/documents/security-considerations-for-code-signing.pdf)
+Occurs when the signing key used is compromised, resulting in a breach
+of trust of the software from the open source community or software
+vendor.  This kind of compromise results in the legitimate software
+being replaced with a malicious, modified version.
+
+- _Mitigation_ - Follow [best practices](https://www.entrustdatacard.com/knowledgebase/best-practices-for-code-signing-certificates)
+regarding code signing key protection.  A more indepth explanation
```
Comment (PushkarJ) [inline supply-chain-security/compromises/compromise-definitions.md:78]:
```suggestion
regarding code signing key protection.  A more in depth explanation
```

**Review by PushkarJ — CHANGES_REQUESTED** [review]

File: supply-chain-security/compromises/compromise-definitions.md (hunk 2 above)
Comment (PushkarJ) [reply supply-chain-security/compromises/compromise-definitions.md:78]: Only for spellcheck reasons

**Comment by lirantal** [comment]
Thank you @PushkarJ for the spelling checks!

**Comment by PushkarJ** [comment]
@lirantal only DCO remaining before we can merge :)

**Comment by lirantal** [comment]
@PushkarJ all ready on my end

**Pull request merged**